npm-shrinkwrap.json

```diff
@@ -262,7 +269,7 @@
       "dependencies": {
         "acorn": {
           "version": "3.3.0",
-          "resolved": "https://registry.npmjs.org/acorn/-/acorn-3.3.0.tgz",
+          "resolved": "http://registry.npmjs.org/acorn/-/acorn-3.3.0.tgz",
```

Comment:
Lots of `http` resolutions, are you using the latest npm?

Reply:
```
$ npm --version
6.4.1
```

Reply:
https://blog.npmjs.org/post/154400916805/avoid-http-urls-in-shrinkwrap-files

for some reason they couldn't fix that on their end 🤷‍♂️

Reply:
Ref: #2641